contrib/utilities/update_scripts/source_files/outdated/update_source_files_to_2.0.0.sed

```diff
@@ -1,6 +1,11 @@
 # A script for the stream editor sed to update .cc and .h files from the
 # naming scheme used in ASPECT 1.5.0 to ASPECT 2.0.0.
 
+# This script is now outdated, because it interferes with valid names in the current
```

Comment:
should we delete it instead?

Reply:
I thought about it, but then decided to just disable and move it instead. Just in case someone still has pre 2.0 plugins out there (e.g. archived plugins of old publications). I think there may be some cases for which they are still useful.